feat: [DATA-2523] Added hyperopt helm chart (#2)
hipagesgroup/chart-hyperopt: README.md

# chart-hyperopt
© Hipages Group Pty Ltd 2019

Helm Chart to deploy [hyperopt](https://github.com/hyperopt/hyperopt): a Distributed Asynchronous Hyperparameter Optimization in Python

> This complements the [hyperkops](https://github.com/hipagesgroup/hyperkops) repository reared towards running hyperopt in Kubernetes. 

Reviewer: ```suggestion
This complements the [hyperkops](https://github.com/hipagesgroup/hyperkops) repository which enables the deployment of hyperopt in Kubernetes. 
```
